Content.Shared/Mind/MindComponent.cs

```diff
@@ -55,6 +55,9 @@ public sealed partial class MindComponent : Component
     [ViewVariables]
     public bool IsVisitingEntity => VisitingEntity != null;
 
+    /// <summary>
+    /// The entity that it's currently visiting. Used for ghosts and the ability to return to body.
```

Comment:
```suggestion
    /// The entity that this mind may be currently visiting. Used, for example, to allow admin ghosting to not make the owner's body catatonic, as opposed to when normally ghosting.
```
this would probably make it more understandable as to what it does